api/src/main/java/net/md_5/bungee/api/event/ChatEvent.java

```diff
@@ -39,4 +41,16 @@ public boolean isCommand()
     {
         return message.length() > 0 && message.charAt( 0 ) == '/';
     }
+
+    /**
+     * Checks whether this message is run on this proxy server
+     *
+     * @return if this command runs on the proxy
+     */
+    public boolean isProxyCommand()
+    {
+        String commandName = message.substring( 1, message.indexOf( " " ) );
```

Comment:
If you use indexOf you need to handle it returning -1 aka missing. (Thought it was clear)

Reply:
oh right, that flew over my head